tests/unit/amo/pages/TestSearchTools.js

```diff
@@ -53,4 +55,43 @@ describe(__filename, () => {
 
     expect(root.find(HeadLinks)).toHaveLength(1);
   });
+
+  describe('enableFeatureRemoveSearchTools = true', () => {
+    const renderWithoutSearchTools = (props = {}) => {
+      const _config = getFakeConfig({ enableFeatureRemoveSearchTools: true });
+
+      return render({ _config, ...props });
+    };
+
+    it('sends a server redirect', () => {
+      const fakeDispatch = sinon.spy(store, 'dispatch');
+
+      renderWithoutSearchTools();
+
+      sinon.assert.calledWith(
+        fakeDispatch,
+        sendServerRedirect({
+          status: 301,
+          url: sinon.match('/en-US/android/search/?category=search-tools'),
+        }),
+      );
+      sinon.assert.callCount(fakeDispatch, 1);
+    });
+  });
+
+  describe('enableFeatureRemoveSearchTools = false', () => {
+    const renderWithSearchTools = (props = {}) => {
```

Comment:
I'm not sure of the value of creating `renderWithSearchTools` when it's only called once. You could probably just inline this code in the test.